packages/account-v2/.eslintignore

```diff
@@ -1,2 +1,4 @@
 dist/*.js
 *.config.*
+*.classnames.ts
```

Comment:
why is this needed?

Reply:
We have a file called classnames.ts which is a convention used for keeping variants styles. We can sort it but I think sorting for that file is not necessary. But this can be taken for discussion